3.6/xenial/hooks/post_push

```diff
@@ -5,4 +5,4 @@ function add_tag() {
     docker push $DOCKER_REPO:$1
 }
 
-add_tag 3.6.1-xenial
+add_tag 3.6.2-xenial
```

Comment:
I wonder if we can someday make this an env var that is automatically set so we only need to update `update.sh`.